changed playermovement to be simplified

RepentMF opened the pull request:
changed PlayerMovement to look cleaner and work more efficiently with future code. also fixes a bug where the player can move while still in an attack animation.

RepentMF committed:
changed playermovement to be simplified

Gavin-C-Greif commented on UtFoM_Project2/Assets/Scripts/PlayerMovement.cs:
```diff
@@ -55,45 +49,10 @@ public void Knock(float kbTime, float damage)
     void Start()
     {
-		currentState = PlayerState.idle;
+		//currentState = PlayerState.idle;
```
Do we want to leave this comment in, or can we delete it?

RepentMF replied:
We can remove it because we are not currently using an in-code state machine.

RepentMF committed:
added stagger/ knockback code to match previous changes

RepentMF committed:
made sword be in the game

RepentMF committed:
added possible future code for equipment and weapons

Gavin-C-Greif committed:
condensed attacking logic

RepentMF committed:
made combo attack for hammer down

gdi unity

RepentMF committed:
reworked playermovement attacks

animations for most attacks still out of center, i THINK sliding while attacking animation is gone?

RepentMF committed:
trying to make knife bug go away

knife will do an extra swing when comboing in a set of even numbers (1, 3, 5, 7, 9, etc all provide the correct response of 1, 3... 9 swings BUT 2, 4, 6, 8, etc all provide responses of 3, 5, 7, 9- an extra swing)

RepentMF committed:
fixed even swing bug for knife

knife1 linking back to knife 2 caused bug- removed animation association

RepentMF committed:
currently working on hitboxes for attacks

RepentMF committed:
added hitboxes to all weapons

-player is not centered in all weapon animations
-sword attack buffer currently broken
-hammer attack buffer sometimes broken

RepentMF committed:
temp fix for attack buffer logic

RepentMF committed:
started adding game rooms and scene transitions

RepentMF committed:
Merge branch 'experimental' of https://github.com/RepentMF/tutorialGame into experimental

RepentMF committed:
started adding the pits map

RepentMF committed:
working on screen transitions

RepentMF committed:
changed camera smoothing in room move

RepentMF committed:
added SS to maps

RepentMF committed:
changed scene transitions

RepentMF committed:
added stuff

RepentMF committed:
added scores

RepentMF committed:
edited stuff in ss

RepentMF committed:
added lots of maps

RepentMF committed:
added maps

RepentMF committed:
changed player hurtbox, edited maps

RepentMF committed:
made lots of changes to combat

RepentMF committed:
edited combat more

RepentMF committed:
edited combat, broke a lot

RepentMF committed:
edited combat

RepentMF committed:
trying to fix what i've wrought

RepentMF committed:
fixed combat some

RepentMF committed:
added status effects and magic

RepentMF committed:
added magics and fixed a bunch of broken stuff

RepentMF committed:
added status stuff and worked on full map

RepentMF committed:
added stuff to fullmap and working on enemy movement speed

RepentMF committed:
worked with more enemy placement

RepentMF committed:
adding stuff to fullmap

RepentMF committed:
map stuff

RepentMF committed:
lots of stuff

RepentMF committed:
created charms

RepentMF committed:
sprite size changes

RepentMF committed:
updated charms

RepentMF committed:
adding treasure chests

RepentMF committed:
adding items and chests to game

RepentMF committed:
changed slime colors

RepentMF committed:
charms

RepentMF committed:
updated map

RepentMF committed:
updated items on map

RepentMF committed:
more items on map

RepentMF committed:
added items

RepentMF committed:
more items

RepentMF committed:
map stuff item stuff

RepentMF committed:
editing map and enemies

RepentMF committed:
enemy logic

RepentMF committed:
enemy logic

RepentMF committed:
enemy stuff

RepentMF committed:
LOTS of enemy stuff

RepentMF committed:
REFACTORED POG!

RepentMF committed:
new stuff

RepentMF committed:
hammerstuff

RepentMF committed:
new animations

RepentMF committed:
new animations

RepentMF committed:
added base inventory

RepentMF committed:
working on voip

RepentMF committed:
made item holder disappear

RepentMF committed:
making new items to put in game world

RepentMF committed:
changing the way player instance is handled- still needs work

RepentMF committed:
added player duplicate fix and inv fix

RepentMF committed:
fixed double knockback on enemy first damage

RepentMF committed:
added new enemy animations

RepentMF committed:
updated feeder and weaver sprites

RepentMF committed:
more sprite work

RepentMF committed:
feeder animations

feeder animations donezo (except devour because no devour logic)

RepentMF committed:
worked on feeder more

added devour

Gavin-C-Greif committed:
updated Inventory.AddItem logic

RepentMF committed:
working on UI

RepentMF committed:
working on ui more

RepentMF committed:
more UI

RepentMF committed:
UI stuff

RepentMF committed:
UI update

RepentMF committed:
changed a LOT of UI stuff

RepentMF committed:
lots of status effect and feeder fixes

RepentMF committed:
worked on top left UI

RepentMF committed:
lots of graphic changes as well as particle systems

RepentMF committed:
attempting to trigger particles on hit

RepentMF committed:
particle system starting to work

RepentMF committed:
particle system stuff

RepentMF committed:
added roommanager

RepentMF committed:
this is really bad code help me

RepentMF committed:
continuing inventory system work

RepentMF committed:
attempting to make inv work

RepentMF committed:
updated items on inkscape map

RepentMF committed:
working on inv

RepentMF committed:
figured out json

RepentMF committed:
items now stay active or inactive between scenes

RepentMF committed:
new hammer stuff

RepentMF committed:
new hammer stuff

RepentMF committed:
updated hammer animation

RepentMF committed:
new hammer animations

RepentMF committed:
hammer stuff

RepentMF committed:
hammer stuff

RepentMF committed:
added a bunch of hammer animations

RepentMF committed:
added a house

RepentMF committed:
added basic lighting

RepentMF committed:
art stuff

RepentMF committed:
feeder light

RepentMF committed:
more lighting stuff

RepentMF committed:
other light changes?

RepentMF committed:
Merge branch 'experimental' of https://github.com/RepentMF/tutorialGame into experimental

RepentMF committed:
lots of stuff

RepentMF committed:
new pause menu

RepentMF committed:
files that aren't game related

RepentMF committed:
Merge branch 'experimental' of https://github.com/RepentMF/tutorialGame into experimental

RepentMF committed:
removed files that aren't from game

RepentMF committed:
sprites updated

RepentMF committed:
working on main menu

RepentMF committed:
adding new things

Gavin-C-Greif committed:
updated gitignore

Gavin-C-Greif committed:
updated files and attached the updated logic to enemies

Gavin-C-Greif committed:
scripted for buildings to fade away while player is within them

Gavin-C-Greif committed:
methods comments

RepentMF committed:
torch animations and walls

Gavin-C-Greif committed:
enemies will appear behind the player when above the player or below the player when at the players feet

Gavin-C-Greif committed:
unity layers update

RepentMF committed:
pause menu and ui for health

RepentMF committed:
feeder changes

RepentMF committed:
Merge pull request #3 from RepentMF/temp/experimental

Temp/experimental
